service/host_test.go

```diff
@@ -0,0 +1,41 @@
+package service
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/evergreen-ci/evergreen"
+	"github.com/evergreen-ci/evergreen/model/host"
+	"github.com/evergreen-ci/evergreen/model/user"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestModifyHostStatus(t *testing.T) {
```

Comment:
Can you append "WithUpdateStatus" to the function name, since this function only tests the "updateStatus"  action